dags/openshift_nightlies/config/benchmarks/openstack.json

```diff
@@ -80,34 +80,32 @@
         {
             "name": "host-network",
             "workload": "network-perf",
-            "command": "./run_hostnetwork_network_test_fromgit.sh test_cloud",
+            "command": "./run.sh",
             "env": {
-                "COMPARE": "false",
-                "ENABLE_SNAPPY_BACKUP": "true",
+                "WORKLOAD": "hostnet",
```

Comment:
The workloads are not configured properly

According to the docs:
```
1. pod to pod using SDN: `WORKLOAD=pod2pod ./run.sh`
2. pod to pod using SDN and network policy: `WORKLOAD=pod2pod NETWORK_POLICY=true ./run.sh`
3. pod to pod using Hostnetwork: `WORKLOAD=pod2pod HOSTNETWORK=true ./run.sh`
4. pod to service: `WORKLOAD=pod2svc ./run.sh`
5. ```

Reply:
done